examples/sap_inventory_update/.devcontainer.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Do not commit codespace config

Reply:
Removed with [1587866](https://github.com/ballerina-platform/module-ballerinax-sap.jco/pull/2/commits/1587866408961566b051f449be7fd7303ce85a9f).